public/js/actions/breakpoints.js

```diff
@@ -127,13 +170,15 @@ function toggleAllBreakpoints(shouldDisableBreakpoints: Boolean) {
 
 /**
  * Update the condition of a breakpoint.
+ *  **NOT IMPLEMENTED**
  *
- * @param object aLocation
+ * @throws {Error} "not implemented"
+ * @memberof actions/breakpoints
+ * @static
+ * @param {Location} location
  *        @see DebuggerController.Breakpoints.addBreakpoint
- * @param string aClients
+ * @param {string} condition
```

Comment:
There are definitely some old docs hanging around.  I think this will shine more light on it to help get it fixed.  

I ran into this at some point and don't know what to make of it https://github.com/devtools-html/debugger.html/blob/master/public/js/selectors.js#L43-L45